src/freenet/clients/http/SecurityLevelsToadlet.java

```diff
@@ -698,10 +726,33 @@ public static void generatePasswordFormPage(boolean wasWrong, HTMLNode formNode,
 			content.addChild("#", l10nSec("enterPassword"));
 		}
 
-		formNode.addChild("input",
-		        new String[] { "type", "name", "size" },
-		        new String[] { "password", "masterPassword", "100" });
-
+		//Creates a table for password prompt and the confirmation box.
+		if(forUpgrade){
```

Comment:
While not an excuse that guideline does not appear to be respected at all here, I will fix all occurrences of this.